client/src/pages/Bookings/NextBookings.tsx

```diff
@@ -0,0 +1,101 @@
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+// import Avatar from '@material-ui/core/ButtonBase';
+import Typography from '@material-ui/core/Typography';
+import { deepOrange } from '@material-ui/core/colors';
+import EditButton from './EditButton';
+
+interface Props {
+  nextBooking: any;
```

Comment:
Maybe use the booking interface here instead of type any?

Ditto throughout where applicable!

Reply:
changed in Integrations issue